nemo/collections/asr/modules/rnnt.py

```diff
@@ -317,20 +315,18 @@ def initialize_state(self, y: torch.Tensor) -> List[torch.Tensor]:
         ]
         return state
 
-    def batch_initialize_states(self, batch_states: List[torch.Tensor], decoder_states: List[List[torch.Tensor]]):
+    def batch_stack_states(self, decoder_states: List[List[torch.Tensor]]):
```

Comment:
Do we need to change the name? Many subclasses depend on this to override

Reply:
Thanks. Restored the original name